theano/gpuarray/basic_ops.py

```diff
@@ -1387,6 +1388,58 @@ def make_node(self, x, axis, splits):
     # we reuse the perform of the CPU op, which is suitable
 
 
+@theano.compile.profiling.register_profiler_printer
+def profile_printer(fct_name, compile_time, fct_call_time, fct_call,
+                    apply_time, apply_cimpl, message, outputs_size,
+                    other_time):
+    if any([x[1].op.__class__.__name__.lower().startswith("gpu")
+            for x in apply_time.keys()]):
+        local_time = sum(apply_time.values())
+        print()
+        print('Some info useful for gpu:')
+
+        cpu = 0
+        gpu = 0
+        trans = 0
+        for (_, node), t in iteritems(apply_time):
+            if isinstance(node.op.__class__.__name__,
+                          (HostFromGpu, GpuFromHost)):
+                trans += t
+            elif node.op.__class__.__name__.lower().startswith("gpu"):
+                gpu += t
+            else:
+                cpu += t
+        print()
+        print("    Spent %.3fs(%.3f%%) in cpu Op, %.3fs(%.3f%%) in gpu Op and %.3fs(%.3f%%) transfert Op" % (
+            cpu, cpu / local_time * 100, gpu, gpu / local_time * 100,
+            trans, trans / local_time * 100))
+
+        print()
+        print("    Theano function input that are float64")
```

Comment:
It is still relevant as most GPU have crappy float64 speed. But maybe we could make this more clear with something like this line:

    Theano function input(s) that are float64. Could cause significant slowdown on some GPUs.

Maybe we should only print them when floatX!=float64?